app/views/streams/show.json.jbuilder

```diff
@@ -9,6 +9,9 @@ json.uploads @stream.uploads do |upload|
   json.files upload.files do |file|
     json.id file.id
     json.filename file.filename
+    json.hash "md5:#{Base64.decode64(file.checksum).unpack1('H*')}"
+    json.type file.content_type
```

Comment:
```suggestion
    json.content_type file.content_type
```

🤷‍♂️ . I'd prefer to keep consistent with the ruby-land method, but `type` feels especially generic (and likely to conflict with JSON-LD or JSON:API if we ever go there..).

Reply:
Yeah -- that makes sense. I was torn about being consistent with the ResourceSync response or being more, uh, descriptive.

Reply:
I think I'll switch `length` to `byte_size` too? so I don't lose my mind